scripts/slither.ts

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+
+import fs from "fs";
+import inquirer from "inquirer";
+import { execSync } from "node:child_process";
+import path from "node:path";
+
+const projectRoot = path.join(__dirname, "../");
+const solcVersion = "0.8.12";
```

Comment:
```suggestion
const solcVersion = "0.8.9";
```

We decided to go for that version, another option is importing it from  `hardhat.config.ts`